### DISCUSSION TITLE
Yi Outdoor 1080p IFUS - no Motion détection at night

### DISCUSSION THREAD

**Discussion by cbroissa-github** [roleoroleo/yi-hack-Allwinner-v2#530] [Q&A]
Hello
I use the integration with mqtt and I have an automation which actives a light when there is a human detection after sunset.
It’s work fine before dusk but at night there is no detection.
It’s the same problem with old release and the last one.
Any idea ?
Thank you for all.

**Comment by roleoroleo** [comment]
Did you configure some schedules with the app?

**Reply by cbroissa-github** [reply]
Yes this

condition: or
conditions:
  - condition: sun
    after: sunset
  - condition: sun
    before: sunrise
